docs/src/components/pricing/PricingTable.tsx

```diff
@@ -635,8 +576,8 @@ const rowHeaders: Record<string, React.ReactNode> = {
   'charts/pie': <ColumnHead label="Pie chart" href="/x/react-charts/pie/" />,
   'charts/sparkline': <ColumnHead label="Sparkline" href="/x/react-charts/sparkline/" />,
   'charts/gauge': <ColumnHead label="Gauge" href="/x/react-charts/gauge/" />,
-  'charts/treemap': <ColumnHead label="Treemap" href="/x/react-charts/treemap/" />,
   'charts/heatmap': <ColumnHead label="Heatmap" href="/x/react-charts/heatmap/" />,
+  'charts/treemap': <ColumnHead label="Treemap" href="/x/react-charts/treemap/" />,
```

Comment:
I appreciated that in the previous positions, it was easily to compare plans, taking along term view. I could see use reverting. Or at least once every feature is built, we should revert.